FRINXio/frinx-frontend
# Handle ipv4 capacity

| Q                        | A 
| ------------------------ | ---
| Fixed Issue?             | `FD-449`
| Tests Added + Pass?      | Yes
| Any Dependency Changes?  | No


In this PR we are fixing the problem when user tried to create pool with 31 or 32 prefixes (ipv4_prefix allocation strategy) and set the subnet to true, then resource manager has problem to correctly calculate the capacity of the newly created resource pool.

Our solution is related also to the fixing PR from the resource-manager itself https://github.com/FRINXio/resource-manager/pull/83 . 

We are solving this problem in UI by disabling the the toggle button when user inputs 31 or 32 prefix.

**How to test it?**
connect to .26 and try to create resource pool of ipv4_prefix strategy and to pool properties enter under prefix 31/32 value and then you will see the subnet toggle disabled
